pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -42,6 +42,8 @@ import (
 	kclientsetinternal "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	kinternalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
 	kadmission "k8s.io/kubernetes/pkg/kubeapiserver/admission"
+	"k8s.io/kubernetes/pkg/quota/generic"
```

Comment:
i think we should fix the imports in this file and split k8s and origin properly, will do after rebase.